src/applications/facility-locator/components/SearchResult.jsx

```diff
@@ -3,7 +3,32 @@ import PropTypes from 'prop-types';
 import LocationInfoBlock from './search-results/LocationInfoBlock';
 import LocationPhoneLink from './search-results/LocationPhoneLink';
 import LocationDirectionsLink from './search-results/LocationDirectionsLink';
+import { CLINIC_URGENTCARE_SERVICE } from '../constants';
 
+const callText = () => (
+  <p>
+    {' '}
+    Before going to a clinic for urgent care, please call the facility to
+    confirm that it's open and able to provide the care you need.{' '}
+  </p>
+);
+
+const urgentCareCall = query => {
+  if (
+    query.facilityType === 'urgent_care' &&
+    query.serviceType === 'NonVAUrgentCare'
+  ) {
+    return callText();
+  }
+  if (
+    query.facilityType === 'cc_provider' &&
+    query.serviceType === CLINIC_URGENTCARE_SERVICE
+  ) {
+    return callText();
+  }
+
+  return null;
+};
```

Comment:
`urgentCareCall` and `callText` to possibly be consolidated into a component of its own -

```jsx
function CallUrgentCare({ query }) {
   const content = (
      <p>
        {' '}
        Before going to a clinic for urgent care, please call the facility to
        confirm that it's open and able to provide the care you need.{' '}
      </p>
  );

if (
    query.facilityType === 'urgent_care' &&
    query.serviceType === 'NonVAUrgentCare'
  ) {
    return content;
  }
  if (
    query.facilityType === 'cc_provider' &&
    query.serviceType === CLINIC_URGENTCARE_SERVICE
  ) {
    return content;
  }

  return null;
}

```

It could also be nice to have the values for `facilityType` as a constant instead of a floating string literal